### DISCUSSION TITLE
Understanding Adjustments

### DISCUSSION THREAD

**Discussion by FlyBoy33** [mriscoc/Ender3V2S1#908] [Q&A]
While this is the best firmware and experience I've ever had, I do have a couple of questions...
When performing the Tramming Wizard, I notice that my probe isn't dropping in the furthest most four corners of my E3V2. How do I adjust the offset so that the CR Touch prob drops closer to the actual corner of my bed? TIA in advance!!!

**Comment by Crazy-Charles** [comment]
The inset is fixed at 30 in the configuration.h file i.e. "#define BED_TRAMMING_INSET_LFRB { 30, 30, 30, 30 } // (mm) Left, Front, Right, Back insets"
However in my custom build version i can change this using a menu on the display.

**Comment by TheWebMachine** [comment]
It's based on bed leveling inset dimensions, which can be modified in the Mesh menu.

If you use the Centered option, this will use 4 corners evenly offset from the edges based on the probe's offset and X/Y Min/Max bounds (set in Probe and Physical Settings menus, respectively).
